resolver/serviceResolver.go

```diff
@@ -335,6 +353,43 @@ func (resolver *serviceResolver) TcsConfig() *core.TcsConfig {
 	}
 }
 
+func (resolver *serviceResolver) checkGuardianAndVerifyCode(userIp string, userAddr string, code string, secondCode string) (*requests.OTPCodeVerifyData, error) {
```

Comment:
```suggestion
func (resolver *serviceResolver) checkGuardianAndVerifyCode(userIp string, request requests.SecurityModeNoExpire) (*requests.OTPCodeVerifyData, error) {
```
gj extracting this into a single method

it can be further simplified after my suggestion to reuse the request payload for both of them. all info is on request

Reply:
updated